components/eamxx/tests/single-process/mam/aero_microphys/output.yaml

```diff
@@ -6,7 +6,10 @@ Fields:
   Physics:
     Field Names:
       - T_mid
-      
+      - oxi_fields
+      - linoz_fields
+      - vertical_emission_fields
+
```

Comment:
This flag will eventually be "off" by default. Will these added fields cause the model to crash if this flag is not defined? If yes, we can remove them after initial evaluation.

Reply:
I commented out these fields and added a note.